src/utilities/borders.scss

```diff
@@ -2,9 +2,6 @@
 // stylelint-disable primer/selector-no-utility
 // stylelint-disable block-opening-brace-space-before, comment-empty-line-before
 
-/* Add a gray border on all edges */
-.border { border: $border !important; }
```

Comment:
This can be removed since it already gets added part of the responsive variants below 👇.